tests/unit/flow-import/flowImport.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {createContainerFlowStack} from '@/store/flow/utils/importHelpers'
+import {IContext} from '@floip/flow-runner'
+
+describe('create_flow_stack', () => {
+  test('should create a flow stack with correct order', () => {
+    const jsonData = `
+      {
+        "flows": [
+          {
+            "uuid": "flow1",
+            "blocks": [
+              {
+                "type": "Core.RunFlow",
+                "config": {
+                  "flow_id": "flow2"
+                }
+              },
+              {
+                "type": "Core.RunFlow",
+                "config": {
+                  "flow_id": "flow4"
+                }
+              }
+            ]
+          },
+          {
+            "uuid": "flow2",
+            "blocks": [
+              {
+                "type": "Core.RunFlow",
+                "config": {
+                  "flow_id": "flow3"
+                }
+              }
+            ]
+          },
+          {
+            "uuid": "flow3",
+            "blocks": []
+          },
+          {
+            "uuid": "flow4",
+            "blocks": []
+          },
+          {
+            "uuid": "flow5",
+            "blocks": []
+          }
+        ]
+      }
+    `
+    const jsonDataParsed = JSON.parse(jsonData)
+
+    const contextObject = {
+    name: 'TODO',
+    description: 'TODO',
+    vendor_metadata: {},
+    flows: jsonDataParsed.flows,
+  } as unknown as IContext
+    const flowStack = createContainerFlowStack(contextObject)
+    expect(flowStack).toEqual(['flow3', 'flow2', 'flow4', 'flow1', 'flow5'])
+  })
```

Comment:
@lakhbawa , as I mentioned this `createContainerFlowStack` is a bit complex, I'd suggest we add another test case to cover "multiple flows importation where there is no dependencies between flows" (I think, now the method would work, but later when someone is updating this method, we gonna need that test case to confirm)

- [ ] add another test case